ci/get-ocp-repo.sh

```diff
@@ -115,24 +120,28 @@ fi
 mkdir -p "$output_dir"
 repo_path="$output_dir/ocp.repo"
 
-set -x
+info "Fetching ${ocp_version} ${rhel_version} repos"
+# set -x
 curl --fail -L "http://base-${ocp_version}-rhel${rhel_version}.ocp.svc.cluster.local" -o "$repo_path"
-set +x
+# set +x
 
 if [ "${rhel_version}" = 96 ]; then
     # XXX: also currently also add 9.4 repos for crun-wasm when building extensions
     # https://github.com/openshift/os/issues/1680
     # https://github.com/openshift/os/pull/1682
     # https://issues.redhat.com/browse/COS-3075
+    info "Fetching 4.19 94 repos"
     curl --fail -L http://base-4-19-rhel94.ocp.svc.cluster.local >> "$repo_path"
 fi
 
-# If we're building the SCOS OKD variant, then strip away all the RHEL repos and just keep the plashet.
+# If we are building the SCOS OKD 9 variant, then strip away all the RHEL repos and just keep the plashet.
 # Temporary workaround until we have all packages for SCOS in CentOS Stream.
-if [ "$osname" = scos ]; then
+# If we are building the SCOS OKD 10 variant, then we need some RHEL packages for now.
+if [ "$osname" = scos ] && [ "${centos_version}" != "10" ]; then
```

Comment:
for the okd-c10s variant `version` here evaluates to `419.96....` because in this specific case it's pulling the version from the default variant (see line 96 above). I pushed a fix. 

This script is hard to maintain :D